sql/moz-fx-data-shared-prod/telemetry_derived/desktop_new_profiles_aggregates_v1/metadata.yaml

```diff
@@ -12,6 +12,7 @@ labels:
   schedule: daily
   depends_on_past: false
   owner1: mhirose
+  table_type: aggregate
```

Comment:
Required?

```suggestion
  shredder_mitigation: true
  table_type: aggregate
```

Reply:
It's actually already above on line 11 (I added this one awhile ago)